PyInstaller/hooks/hook-sqlalchemy.py

```diff
@@ -24,6 +24,9 @@
 # are not. We should explicitly include database backends.
 hiddenimports = ['pysqlite2', 'MySQLdb', 'psycopg2']
 
+# add sqlalchemy.ext.baked to hidden imports
+hiddenimports.append("sqlalchemy.ext.baked")
```

Comment:
Please add this to the hiddenimports list definition above.
```suggestion
hiddenimports = ['pysqlite2', 'MySQLdb', 'psycopg2', 'sqlalchemy.ext.baked']
```